sqlmesh/core/snapshot/evaluator.py

```diff
@@ -410,6 +410,7 @@ def _create_snapshot(
             engine_adapter=self.adapter,
             snapshots=parent_snapshots_by_name,
             is_dev=is_dev,
+            execute=True,
```

Comment:
wait why is this one True? shouldn't it be false? this is the section where we create the table

Reply:
Yeah this was an oversight, thanks for catching